app/src/main/java/edu/uc/mille5bm/smartnutrition/CalorieCounterActivity.kt

```diff
@@ -25,11 +25,12 @@ class CalorieCounterActivity: AppCompatActivity() {
     }
     fun submit(v: View?) {
         findViewById<View>(R.id.llAddActivity).visibility = View.GONE
-        var ed1 = findViewById<EditText>(R.id.etName)
-        val ed2 = findViewById<EditText>(R.id.etCal)
+        var edName = findViewById<EditText>(R.id.etName)
+        val edCalorie = findViewById<EditText>(R.id.etCal)
         //listItems.add(ed1.text.toString() + ",  Calories Burned: " + ed2.text)
       //  adapter!!.notifyDataSetChanged()
-        ed1.setText("")
-        ed2.setText("")
+        //ed1 and ed2 were unclear for variable names.
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.